assets/js/components/HostDetails/HostChecksSelection.jsx

```diff
@@ -49,7 +49,9 @@ function HostChecksSelection({
               type="primary"
               className="mx-1"
               onClick={() => {}}
-              disabled={!canStartExecution(selectedChecks, isSavingSelection)}
+              disabled={
+                !canStartExecution(hostSelectedChecks, isSavingSelection)
+              }
```

Comment:
I think you can move this check outside and avoid passing the `hostSelectedChecks` prop

Reply:
Lets try :+1: